app/javascript/spec/ansible-playbook-workflow/ansible-playbook-workflow.data.js

```diff
@@ -1,15 +1,6 @@
-export const ansiblePlaybookData = {
+/* eslint-disable max-len */
+export const ansiblePlaybookData = JSON.stringify({
   id: 1000,
-  payload: {
-    name: 'John Doe',
-    age: 30,
-    email: 'johndoe@example.com',
-    address: {
-      street: '123 Main St',
-      city: 'Anytown',
-      zip: 12345,
-    },
-    hobbies: ['Reading', 'Hiking', 'Cooking'],
-  },
+  payload: "---\n- name: Provision instances\n hosts: localhost\n connection: local\n gather_facts: False\n\n # load AWS variables from this group vars file\n vars_files:\n - group_vars/all\n\n tasks:\n - name: Launch instances\n ec2:\n access_key: \"{{ ec2_access_key }}\"\n secret_key: \"{{ ec2_secret_key }}\"\n keypair: \"{{ ec2_keypair }}\"\n group: \"{{ ec2_security_group }}\"\n type: \"{{ ec2_instance_type }}\"\n image: \"{{ ec2_image }}\"\n region: \"{{ ec2_region }}\"\n instance_tags: \"{'ansible_group':'jboss', 'type':'{{ ec2_instance_type }}', 'group':'{{ ec2_security_group }}', 'Name':'demo_''{{ tower_user_name }}'}\"\n count: \"{{ ec2_instance_count }}\"\n wait: true\n register: ec2\n\n - name: Wait for SSH to come up\n wait_for:\n host: \"{{ item.public_dns_name }}\"\n port: 22\n delay: 60\n timeout: 320\n state: started\n with_items: \"{{ ec2.instances }}\"\n",
```

Comment:
Super-minor, but for readability I think you can use a template literal:

```suggestion
  payload: `
---
- name: Provision instances
  hosts: localhost
  connection: local
  gather_facts: False

  # load AWS variables from this group vars file
  vars_files:
    - group_vars/all

  tasks:
    - name: Launch instances
      ec2:
      access_key: "{{ ec2_access_key }}"
      secret_key: "{{ ec2_secret_key }}"
      keypair: "{{ ec2_keypair }}"
      group: "{{ ec2_security_group }}"
      type: "{{ ec2_instance_type }}"
      image: "{{ ec2_image }}"
      region: "{{ ec2_region }}"
      instance_tags: "{'ansible_group':'jboss', 'type':'{{ ec2_instance_type }}', 'group':'{{ ec2_security_group }}', 'Name':'demo_''{{ tower_user_name }}'}"
      count: "{{ ec2_instance_count }}"
      wait: true
      register: ec2

    - name: Wait for SSH to come up
      wait_for:
      host: "{{ item.public_dns_name }}"
      port: 22
      delay: 60
      timeout: 320
      state: started
      with_items: "{{ ec2.instances }}"`,
```